### PR TITLE
Device: added parallel make() function that takes in string vector

### PR DESCRIPTION
**Pull request by ncorgan**
* Bumped API and ABI to 0.8.1

### CODE REVIEW

File: include/SoapySDR/Version.h [hunk 1]
```diff
@@ -35,7 +35,7 @@
  * And <i>extra</i> is empty for releases but set on development branches.
  * The ABI should remain constant across patch releases of the library.
  */
-#define SOAPY_SDR_ABI_VERSION "0.8"
+#define SOAPY_SDR_ABI_VERSION "0.8-1"
```
Comment (guruofquality) [inline include/SoapySDR/Version.h:38]: The internet people are going to come after me with pitchforks and torches. I didnt tag it yet. Just throw a line in the changelog. Its not really breaking anything. I'll merge this before the tag.

Comment (guruofquality) [reply include/SoapySDR/Version.h:38]: I mean dont change the magic numbers.

Comment (ncorgan) [reply include/SoapySDR/Version.h]: Done

**Comment by guruofquality** [comment]
Bit of a new warning for that new function, we finally confused swig:
```
[ 94%] Swig compile SoapySDR.i for python
/home/src/SoapySDR/include/SoapySDR/Device.hpp:105: Warning 509: Overloaded method SoapySDR::Device::make(std::vector< std::string,std::allocator< std::string > > const &) effectively ignored,
/home/src/SoapySDR/include/SoapySDR/Device.hpp:95: Warning 509: as it is shadowed by SoapySDR::Device::make(SoapySDR::KwargsList const &).
[ 94%] Built target SoapySDR_swig_compilation
```

**Pull request merged**